python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -1290,7 +1290,7 @@ def _impl_v18(cls, bb, inputs, attr, params):
         ), "Only one of scales and sizes can be provided in Resize."
 
         # Define relax implementation.
-        if roi is not None:
+        if roi is not None and not _np.array_equal(roi.data.numpy(),  _np.array([], dtype=_np.float32)):
```

Comment:
It's not robust to use `equal` check, as the dtype can be different. Please check if it's a 0-ranked tensor

Reply:
Okay, I'll try